src/arch/ucx/machine-onesided.C

```diff
@@ -131,12 +142,9 @@ void LrtsIssueRget(NcpyOperationInfo *ncpyOpInfo)
     } else {
         // Remote buffer is not registered, ask peer to perform put
```

Comment:
minor: this comment is not relevant anymore

Reply:
Good point, fixed in the latest push.